code/__DEFINES/status_effects.dm

```diff
@@ -55,7 +55,7 @@
 
 //#define STATUS_EFFECT_HISWRATH /datum/status_effect/his_wrath //His Wrath.
 
-#define STATUS_EFFECT_SUMMONEDGHOST /datum/status_effect/cultghost //is a cult ghost and can't use manifest runes
+#define STATUS_EFFECT_SUMMONEDGHOST /datum/status_effect/cultghost //is a cult ghost : can see dead people, cant manifest more ghosts
```

Comment:
on one hand cult ghosts being able to see other ghosts is moot as they are pulled from there. on the other hand...possible meta potional though really i may be over worrying as these things are just cannon fodder.

Reply:
what 'meta potential'? cultists can directly interact with ghosts and have access to all dchat knowledge anyway, its not OOC knowledge. this is so that a dark spirit can direct cult ghosts to where they need to go